src/EditorFeatures/CSharp/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -0,0 +1,181 @@
+                                          IDEDiagnosticIds.PreferFrameworkTypeInMemberAccessDiagnosticId)
+                },
+                {
+                    FeatureOnOffOptions.FixAddRemoveBraces,
+                    ImmutableArray.Create(IDEDiagnosticIds.AddBracesDiagnosticId)
+                },
+                {
+                    FeatureOnOffOptions.FixAccessibilityModifiers,
+                    ImmutableArray.Create(IDEDiagnosticIds.AddAccessibilityModifiersDiagnosticId)
+                },
+                {
+                    FeatureOnOffOptions.SortAccessibilityModifiers,
+                    ImmutableArray.Create(IDEDiagnosticIds.OrderModifiersDiagnosticId)
+                },
+                {
+                    FeatureOnOffOptions.MakeReadonly,
+                    ImmutableArray.Create(IDEDiagnosticIds.MakeFieldReadonlyDiagnosticId)
+                },
+                {
+                    FeatureOnOffOptions.RemoveUnnecessaryCasts,
+                    ImmutableArray.Create(IDEDiagnosticIds.RemoveUnnecessaryCastDiagnosticId)
+                },
+                {
+                    FeatureOnOffOptions.FixExpressionBodiedMembers,
+                    ImmutableArray.Create(IDEDiagnosticIds.UseExpressionBodyForConstructorsDiagnosticId,
+                                          IDEDiagnosticIds.UseExpressionBodyForMethodsDiagnosticId,
+                                          IDEDiagnosticIds.UseExpressionBodyForConversionOperatorsDiagnosticId,
+                                          IDEDiagnosticIds.UseExpressionBodyForOperatorsDiagnosticId,
+                                          IDEDiagnosticIds.UseExpressionBodyForPropertiesDiagnosticId,
+                                          IDEDiagnosticIds.UseExpressionBodyForIndexersDiagnosticId,
+                                          IDEDiagnosticIds.UseExpressionBodyForAccessorsDiagnosticId)
+                },
+                {
+                    FeatureOnOffOptions.FixInlineVariableDeclarations,
+                    ImmutableArray.Create(IDEDiagnosticIds.InlineDeclarationDiagnosticId)
+                },
+                {
+                    FeatureOnOffOptions.RemoveUnusedVariables,
+                    ImmutableArray.Create(IDEDiagnosticIds.DeclaredVariableNeverUsedDiagnosticId,
+                                          IDEDiagnosticIds.AssignedVariableNeverUsedDiagnosticId)
+                },
+                {
+                    FeatureOnOffOptions.FixObjectCollectionInitialization,
+                    ImmutableArray.Create(IDEDiagnosticIds.UseObjectInitializerDiagnosticId,
+                                          IDEDiagnosticIds.UseCollectionInitializerDiagnosticId)
+                }
+            };
+
+            return dictionary.ToImmutableDictionary();
+        }
+
+        public async Task<Document> CleanupDocument(Document document, CancellationToken cancellationToken)
```

Comment:
Task returning methods should end in "Async"

Reply:
fixed